pkgs/tools/misc/starfetch/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "starfetch";
+  version = "unstable-2022-04-06";
+
+  src = fetchFromGitHub {
+    owner = "Haruno19";
+    repo = "starfetch";
+    rev = "7d648f32226b35c6bd41ea88425fbf779d72367e";
+    sha256 = "sha256-G/PqALWIjRJWT8jrOK/dOBX98W5y9i1mXzjZVWzz5RQ=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp starfetch $out/bin/
+  '';
+
+  meta = with lib; {
+    description = "CLI star constellations displayer";
+    homepage = "https://github.com/Haruno19/starfetch";
+    license = licenses.gpl3Only;
```

Comment:
Maybe double check this is Only and not Plus

https://github.com/Haruno19/starfetch/blob/main/LICENSE#L640